tuf/repository_tool.py

```diff
@@ -2754,6 +2755,44 @@ def delegations(self):
 
 
 
+
+  def _check_path(self, pathname):
+    """
+    <Purpose>
+      Check if a path matches the definition of a PATHPATTERN or a
+      TARGETPATH (uses the forward slash (/) as directory separator and
+      does not start with a directory separator). Checks are performed only
+      on the path string, without accessing the file system.
+
+    <Arguments>
+      pathname:
+        A file path or a glob pattern.
+
+    <Exceptions>
+      securesystemslib.exceptions.FormatError, if 'pathname' is improperly
+      formatted.
+
+      tuf.exceptions.InvalidNameError, if 'pathname' does not match pattern.
+
+    <Returns>
+      None.
+    """
+
+    tuf.formats.RELPATH_SCHEMA.check_match(pathname)
+
+    if '\\' in pathname:
+      raise tuf.exceptions.InvalidNameError('Path ' + repr(pathname)
+          + ' does not use the forward slash (/) as directory separator.')
+
+    if pathname.startswith(os.sep):
```

Comment:
I'm pretty sure that this would allow a windows user to pass an absolute path that starts with "/", which we also don't want. I suggest to change to `if pathname.startswith('/'):`. Or am I missing something?

Reply:
See comment below